edib/make-app.mk

```diff
@@ -27,7 +27,7 @@ postinfo:
 release: $(RELEASE_FILE)
 
 $(RELEASE_FILE): app-compile phoenix-assets
-	$(IN_APP_DIR) MIX_ENV=$(MIX_ENV) mix release
+	$(IN_APP_DIR) mix release --env=prod
```

Comment:
We probably still need to support custom provided envs, therefore it should be changed to:

```
$(IN_APP_DIR) mix release --env=$(MIX_ENV:-prod)
```

For the future we can think about dividing the mix and the release env, but for now we should stick to what user might expect currently.

WDYT?